### PR TITLE
Create Contributing README

### PR DESCRIPTION
**Pull request by lauterbur**
Some of this information may change depending on final decisions from Andy and Jerome (installation? jupyter? fork?). Want something up ASAP so folks can get started installing.

### CODE REVIEW

File: adding_species/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Important!
+Install stdpopsim following the instructions here:
+https://stdpopsim.readthedocs.io/en/latest/development.html#installation
+You may also need the other packages listed in the [environment.yml](https://github.com/popsim-consortium/workshops/blob/main/environment.yml).
+
+If you run into any problems, ask for help in the #workshops or #newbie-help channels on Slack (popsimgroup.slack.com).
+
+At least 15 minutes before the workshop:
+1. Click the JupyterHub link (TBA)
```
Comment (petrelharp) [inline adding_species/README.md:13]: no jupyterhub

**Comment by petrelharp** [comment]
Looks good other than the TBA github link and the jupyterhub mention.

**Comment by lauterbur** [comment]
Was Andy making a fork for folks to clone? I don't remember what was decided this morning.

**Comment by andrewkern** [comment]
no i'm still working on the materials @lauterbur 😬

**Comment by lauterbur** [comment]
Thanks. I'd like to send the instructions email out linking this asap even if all the links and materials aren't quite available.

**Comment by andrewkern** [comment]
Hey @lauterbur I'm purposefully giving some overlap between what you have here and the workshop materials at #25 just FYI

**Pull request merged**